resources/views/packs/index.blade.php

```diff
@@ -27,8 +27,7 @@
                         <div class="beatmap-pack__name">{{ $pack->name }}</div>
                         <div class="beatmap-pack__details">
                             <span class="beatmap-pack__date">{{ json_date($pack->date) }}</span>
-                            <span class="beatmap-pack__author">by </span>
-                            <span class="beatmap-pack__author beatmap-pack__author--bold">{{ $pack->author }}</span>
+                            <span>{!! osu_trans('beatmappacks.show.created_by', ['author' => "<strong>{make_blade_safe($pack->author)}</strong>"]) !!}</span>
```

Comment:
This is just putting the function call into the string....

Also looking at it again, it can just use `e` instead of `make_blade_safe`/wrapping in `HtmlString` since it's being used in the blade itself and not being passed anywhere; just use the `tag` helper `tag('stong', content: e($pack->author))`